bv-evaluation/plot.py

```diff
@@ -809,9 +843,9 @@ def scatter_solving_time_instcombine(df):
     time_min = min(df["solved_bitwuzla_times_average"].min(), tot_sum.min())
     time_max = max(df["solved_bitwuzla_times_average"].max(), tot_sum.max())
 
-    ax.set_ylim(
-        10 ** (np.floor(np.log10(time_min))), 10 ** (np.ceil(np.log10(time_max)))
-    )
+    # ax.set_ylim(
+    #     10 ** (np.floor(np.log10(time_min))), 10 ** (np.ceil(np.log10(time_max)))
+    # )
```

Comment:
We should ensure that all our plots have the same *x* and *y* scales. Could we do this please?

Reply:
this comment was relative to another plot, anyways I uncommented it and set all the plots to the same scale.